test/utils/page_objects/base_page/filter.js

```diff
@@ -0,0 +1,19 @@
+const Collection = require('../base_elements/base_collection');
+
+class Filter extends Collection {
+    constructor() {
+        super();
+        this.filterSizeValues = new Collection('className', 'select-box__option ng-star-inserted');
+        this.filterColorMenu = new Collection('className', 'dsw-accordion__label'); 
+        this.filterColorValues = new Collection('xpath', '//div[@class="exposed-filter__checkbox__label"]/span[text()="Black"]');
+        this.filterResult = new Collection('className', 'active-filters__filter ng-star-inserted');
```

Comment:
You can use all these filters as the corresponding properties of the corresponding page. See https://github.com/XenaF/MODULE_7/pull/2.